### PR TITLE
add logging statements to track process

### PR DESCRIPTION
**Pull request by kba**
forgot to commit/push this one.

### CODE REVIEW

File: ocrd_olahd_client/processor.py [hunk 1]
```diff
@@ -21,12 +22,18 @@ def __init__(self, *args, **kwargs):
...
         bagger = WorkspaceBagger(Resolver(), strict=True)
         # TODO
         dest = join(gettempdir(), 'bag-%d.ocrd.zip' % int(round((time() * 1000))))
         # TODO
         ocrd_identifier = self.workspace.mets.unique_identifier
+        LOG.info('Bagging workspace')
         bagger.bag(self.workspace, ocrd_identifier, dest=dest)
+        LOG.info('Logging in')
         client.login()
+        LOG.info('POST bag')
```
Comment (joschrew) [inline ocrd_olahd_client/processor.py:37]:
```suggestion
        LOG.debug('Bagging workspace')
        bagger.bag(self.workspace, ocrd_identifier, dest=dest)
        LOG.debug('Logging in')
        client.login()
        LOG.debug('POST bag')
```

Comment (joschrew) [reply ocrd_olahd_client/processor.py:37]: Personally I prefer not to log too much to not make the code harder to read. I don't read logs often (except error logs) and prefer to debug in those cases. I am ok with it tough but would rather remove it or set parts of it to debug.

**Review by joschrew — COMMENTED** [review]
placeholder

File: ocrd_olahd_client/processor.py (hunk 1 above)
Comment (kba) [reply ocrd_olahd_client/processor.py]: Less noise is better, agreed.

**Pull request merged**